### PR TITLE
[-] CORE : #PSCSX-4339 - "default" theme name should be customizable

### PR DESCRIPTION
**Pull request by guestisp**
Fix #PSCSX-4339 - "default" theme name should be customizable

### CODE REVIEW

File: classes/controller/AdminController.php [hunk 1]
```diff
@@ -313,9 +313,14 @@ public function __construct()
 		if ($this->multishop_context == -1)
 			$this->multishop_context = Shop::CONTEXT_ALL | Shop::CONTEXT_GROUP | Shop::CONTEXT_SHOP;
 
-		$this->bo_theme = ((Validate::isLoadedObject($this->context->employee) && $this->context->employee->bo_theme) ? $this->context->employee->bo_theme : 'default');
+        // Fix #PSCSX-4339 - "default" theme name should be customizable
```
Comment (gRoussac) [inline classes/controller/AdminController.php]: Hi,

thank you for your PRs.
Can you please use tabs as indentation ?

Regards

Comment (guestisp) [inline classes/controller/AdminController.php]: Sure, sorry for that, i'm using a temporary laptop
i'm not on my desktop.

Comment (guestisp) [inline classes/controller/AdminController.php]: Can you merge this PR and fix indentation?
I'm unable to use tab from this laptop

**Comment by jeromenadaud** [comment]
Hi,

Thank you for your PR.

Best regards.

**Comment by guestisp** [comment]
Why have you replaced my `file_exists` with `@filemtime` ? I don't like error suppression with `@`
Any advantage with this ?

**Comment by gRoussac** [comment]
Hi,

http://blog.rodneyrehm.de/archives/12-Improving-Disk-IO-in-PHP-Apps.html

Regards

**Comment by guestisp** [comment]
Good to know, thank you.

**Pull request merged**